spec/visitorSpec.js

```diff
@@ -0,0 +1,20 @@
+visit = require("../src/visitor");
+var fs = require('fs');
+
+describe("spies", function () {
+  var visitor = new Visitor("bob", "smith", 2 / 1 / 2019, 7, "sally");
```

Comment:
I can tell you that these test don't run against your project for 2 reason
1. Your Visitor class is not export please add this to you class 
```module.exports = Visitor;```
2. Your require statement points to [ require("../src/visitor"); ] and your Visitor class is not in src so jasmine will not know where to find the class.